language/polkavm/move-to-polka/src/stackless/llvm.rs

```diff
@@ -1671,8 +1671,7 @@ unsafe fn add_polkavm_metadata(
     let mut struct_field_types = [array_ty];
     let struct_ty = LLVMStructType(struct_field_types.as_mut_ptr(), 1, 0);
     let text = CString::new(fn_name).unwrap();
-    let const_array =
-        LLVMConstStringInContext2(context, text.as_ptr(), text.as_bytes().len() as usize, 1);
+    let const_array = LLVMConstStringInContext2(context, text.as_ptr(), text.as_bytes().len(), 1);
```

Comment:
Future improvement in another PR.

Maybe we should try to document that 1 here means **_DontNullTerminate_** [doc](https://llvm.org/doxygen/group__LLVMCCoreValueConstantComposite.html)

Maybe something like this

```
const DONT_NULL_TERMINATE: u32 = 1;
let const_array = LLVMConstStringInContext2(context, text.as_ptr(), text.as_bytes().len(), DONT_NULL_TERMINATE);
```